### PR TITLE
fix: paycheck vulnerability

### PR DESCRIPTION
**Pull request by JamesEU**
It Fixes the known paycheck vulnerability people exploit within vRP.

### CODE REVIEW

File: vrp/modules/paycheck.lua [hunk 1]
```diff
@@ -1,133 +1,145 @@
 local Proxy = module("vrp", "lib/Proxy")
```
Comment (arthur-rl) [inline vrp/modules/paycheck.lua:1]: You don't need this line. Please remove.

**Review by arthur-rl — CHANGES_REQUESTED** [review]
Please remove the line I commented on thanks.

File: vrp/modules/paycheck.lua (hunk 1 above)
Comment (JamesEU) [reply vrp/modules/paycheck.lua:1]: Was in default so just left it but will do

**Comment by JamesEU** [comment]
> Please remove the line I commented on thanks.

Done 👍

**Pull request closed without merging**